functional_tests/scylla_operator/libs/helpers.py

```diff
@@ -41,6 +41,14 @@ class PodStatuses(Enum):
     CRASH_LOOP_BACK_OFF = 'CrashLoopBackOff'
 
 
+def create_namespace(db_cluster: ScyllaPodCluster, namespace: str) -> None:
```

Comment:
don't we have this help code already somewhere inside the k8s cluster implementation ?

Reply:
after reading the rest of the PR, I can guess why we need a separate implementation of it.
but we should still write it down, why we have this separated from the k8s cluster

Reply:
I can add it as method for the K8S cluster.